google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/main/java/com/google/cloud/spanner/jdbc/SpannerPool.java

```diff
@@ -211,7 +211,7 @@ Spanner getSpanner(ConnectionOptions options, ConnectionImpl connection) {
       if (!initialized) {
         initialize();
       }
-      if (spanners.containsKey(key)) {
+      if (spanners.get(key) == null) {
```

Comment:
I think this is backwards now?

Reply:
can we add some more testing around this?

Reply:
Oops, that's indeed backward. It's already covered by the test case [here](https://github.com/googleapis/google-cloud-java/blob/a070c22624a73246059e3b694af6d4595de62fc4/google-cloud-clients/google-cloud-contrib/google-cloud-spanner-jdbc/src/test/java/com/google/cloud/spanner/jdbc/SpannerPoolTest.java#L93), but because this PR has build errors, I didn't pick up on that one. I added the same error to the private repo and ran the test cases, and there were multiple test failures.